package.json

```diff
@@ -22,7 +22,8 @@
     "babel-preset-env": "^1.7.0",
     "babel-preset-es2015": "^6.24.1",
     "babel-register": "^6.26.0",
-    "browser-sync": "^2.23.6",
+    "browser-sync": "^2.24.7",
+    "create-cert": "^1.0.6",
```

Comment:
I think we're missing `"generate-ssl-cert": "gulp generateCert",` under the scripts section.

And maybe we can change it to just be `generateCert` so: `"generateCert": "gulp generateCert",`

Reply:
So then the instructions have to be changed to run `npm run generateCert`.

Reply:
Good catch, I missed the `genrateCert` command being lost in resolving the merge conflict. I added it back with the new name `generateCert` and updated the README.

Reply:
Thanks!